package.json

```diff
@@ -1080,6 +1080,7 @@
     "regenerator-runtime": "^0.13.9"
   },
   "dependencies": {
+    "@folio/stripes-acq-components": "^4.0.200000469",
```

Comment:
1. `*200000469` do now include such version
2. do not add `stripes-acq-components` to ui-users 
3. for the future, such packages should be included as stripesDep